src/solver/ts-generator/hydro.cpp

```diff
@@ -311,19 +314,23 @@ bool GenerateHydroTimeSeries(Data::Study& study, uint currentYear)
             logs.info() << "Archiving the hydro time-series";
             String output;
             study.areas.each([&](const Data::Area& area) {
-                study.buffer.clear() << study.folderOutput << SEP << "ts-generator" << SEP
-                                     << "hydro" << SEP << "mc-" << currentYear << SEP << area.id;
-                if (IO::Directory::Create(study.buffer))
+                study.buffer.clear() << "ts-generator" << SEP << "hydro" << SEP << "mc-"
+                                     << currentYear << SEP << area.id;
+
                 {
+                    std::string ror_buffer;
                     output.clear() << study.buffer << SEP << "ror.txt";
-                    area.hydro.series->ror.saveToCSVFile(output);
+                    writer->addJob(output.c_str(), ror_buffer);
```

Comment:
I don't understand how the following instruction can work : 
```c++
writer->addJob(output.c_str(), ror_buffer);
```
`area.hydro.series->ror` should turned into a buffer, and then passed to **addJob**